File: CSharp/k02_3.cs
using System;

namespace CSharp
{
    public class K02_3 : IExecutable
    {
        public void Execute()
        {
            try
            {
                Console.Write("一つ目の商品の値段は？");
                int priceA = int.Parse(Console.ReadLine());
                Console.Write("個数は？");
                int amountA = int.Parse(Console.ReadLine());
                Console.Write("二つ目の商品の値段は？");
                int priceB = int.Parse(Console.ReadLine());
                Console.Write("個数は？");
                int amountB = int.Parse(Console.ReadLine());

                Console.WriteLine(string.Format("お支払いは税込み￥{0}です", (int)((priceA * amountA + priceB * amountB) * 1.08f)));

            }
            catch (Exception ex)
            {
                Console.WriteLine(ex.Message);
            }
        }
    }
}

File: Python/k02_3.py
class k02_3(object):
    
    def execute(self):
        try:
            print('一つ目の商品の値段は？')
            priceA = int(input())
            print('個数は？')
            amountA = int(input())
            print('二つ目の商品の値段は？')
            priceB = int(input())
            print('個数は？')
            amountB = int(input())

            total = (priceA * amountA + priceB * amountB) * 1.08
            print('お支払いは税込み￥{}です'.format(total))
        except ValueError as ve:
            print(ve)


